lib/db/schema/patch-080-079.sql

```diff
@@ -0,0 +1,28 @@
+-- SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+-- alter table recoveryCodes add createdAt BIGINT UNSIGNED NOT NULL;
+
+-- CREATE PROCEDURE `createRecoveryCode_1` (
```

Comment:
These create procedures can be replace with a single drop `DROP PROCEDURE createRecoveryCode_3`. Technically, by dropping this procedure and adding back the column, the database will be in the same state it was before.